haskell/src/Monocle/Backend/Queries.hs

```diff
@@ -1014,3 +1014,18 @@ getActivityStats = do
               . length
               $ haBuckets hbSubBuckets
        in SearchPB.Histo {..}
+
+getSuggestions :: QueryM SearchPB.SuggestionsResponse
+getSuggestions = do
+  suggestionsResponseTaskTypes <- getTop "tasks_data.ttype"
+  suggestionsResponseAuthors <- getTop "author.muid"
+  suggestionsResponseApprovals <- getTop "approval"
+  suggestionsResponsePriorities <- getTop "tasks_data.priority"
+  suggestionsResponseSeverities <- getTop "tasks_data.severity"
+  let suggestionsResponseProjects = mempty
+  let suggestionsResponseGroups = mempty
```

Comment:
```suggestion
getSuggestions :: Config.Index -> QueryM SearchPB.SuggestionsResponse
getSuggestions config = do
  suggestionsResponseTaskTypes <- getTop "tasks_data.ttype"
  suggestionsResponseAuthors <- getTop "author.muid"
  suggestionsResponseApprovals <- getTop "approval"
  suggestionsResponsePriorities <- getTop "tasks_data.priority"
  suggestionsResponseSeverities <- getTop "tasks_data.severity"
  let suggestionsResponseProjects = Config.getTenantProjectsNames config
  let suggestionsResponseGroups = fst <$> Config.getTenantGroups config
```

Reply:
well why not, I'll try your proposal

Reply:
Oops I missed the projects and groups were added afterward. Well It looks better to do it all at once. Thanks!